sdk/nodejs/src/environmentConfigManager.ts

```diff
@@ -80,10 +81,10 @@ export class EnvironmentConfigManager {
 
         if (res?.status === 304) {
             this.logger.debug(`Config not modified, using cache, etag: ${this.configEtag}`)
-        } else if (res?.status === 200 && res?.data) {
-            getBucketingLib().setConfigData(this.environmentKey, JSON.stringify(res.data))
+        } else if (res?.status === 200 && projectConfig) {
+            getBucketingLib().setConfigData(this.environmentKey, JSON.stringify(projectConfig))
```

Comment:
Optimization here, can we not call `res.json()` and just pass in `res.text()`?

Reply:
technically yes but currently we rely on the JSON parsing exception that gets thrown by parsing this response. If you pass a non-parseable string into the bucketing lib it's a different error that's a bit less specific